src/blocks/remote-data-html/editor.scss

```diff
@@ -0,0 +1,14 @@
+// Styled after core/html. This overlay will allow clicking into the block
+// when HTML is present and bypass iframe bubbling issues.
```

Comment:
accurate?
```suggestion
// when HTML is present and intercept all mouse events.
```

Reply:
That's correct, updated and expanded in https://github.com/Automattic/remote-data-blocks/pull/314/commits/ff57a4db2069e9feb27a8546f158a45497dcc214!